app/packages/plugins/src/index.ts

```diff
@@ -15,11 +15,13 @@ declare global {
   }
 }
 
-// required for plugins to use the same instance of React
-window.React = React;
-window.ReactDOM = ReactDOM;
-window.recoil = recoil;
-window.__fos__ = fos;
+if (typeof window !== "undefined") {
```

Comment:
Unrelated to PR, but something I plan to do is add `components` here as well, hopefully thats compatible with SSR